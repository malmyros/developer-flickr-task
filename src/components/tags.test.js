import React from 'react';
import { shallow } from 'enzyme';
import Tags from './tags';

describe('tag', () => {
  it('should render without crashing', () => {
    const tags = [
      { id: 1, title: 'tag 1' },
      { id: 2, title: 'tag 2' },
      { id: 3, title: 'tag 3' },
    ];

    const component = shallow(<Tags tags={tags}/>);
    expect(component.find('Tag')).toBeTruthy();
    expect(component.find('Tag')).toHaveLength(3);
    expect(component).toMatchSnapshot();
  });
});
