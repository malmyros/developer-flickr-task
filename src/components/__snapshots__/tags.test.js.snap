// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tag should render without crashing 1`] = `
<Fragment>
  Tags: 
  <Tag
    key="tag_id_tag.id"
    title=""
  />
  <Tag
    key="tag_id_tag.id"
    title=""
  />
  <Tag
    key="tag_id_tag.id"
    title=""
  />
</Fragment>
`;
